components/style/servo/selector_parser.rs

```diff
@@ -100,44 +100,55 @@ impl PseudoElement {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[allow(missing_docs)]
 pub enum NonTSPseudoClass {
-    AnyLink,
-    Link,
-    Visited,
     Active,
+    AnyLink,
+    Checked,
+    Disabled,
+    Enabled,
     Focus,
     Fullscreen,
     Hover,
-    Enabled,
-    Disabled,
-    Checked,
     Indeterminate,
-    ServoNonZeroBorder,
+    Lang(String),
```

Comment:
I still believe this should be an atom, but if you don't agree, let's at least use `Lang(Box<str>)`, so the size increment is not so huge.